src/sql/src/parsers/alter_parser.rs

```diff
@@ -222,12 +199,43 @@ impl ParserContext<'_> {
         }
     }
 
-    fn parse_alter_table_drop_default(&mut self) -> Result<AlterTableOperation> {
-        let columns = self
-            .parser
-            .parse_comma_separated(parse_alter_column_drop_default)
-            .context(error::SyntaxSnafu)?;
-        Ok(AlterTableOperation::DropDefaults { columns })
+    fn parse_alter_table_drop_default(
+        &mut self,
+        column_name: Ident,
+    ) -> Result<AlterTableOperation> {
+        let drop_default = DropDefaultsOperation(column_name);
+        if self.parser.consume_token(&Token::Comma) {
+            let mut columns = self
+                .parser
+                .parse_comma_separated(parse_alter_column_drop_default)
+                .context(error::SyntaxSnafu)?;
+            columns.insert(0, drop_default);
+            Ok(AlterTableOperation::DropDefaults { columns })
+        } else {
+            Ok(AlterTableOperation::DropDefaults {
+                columns: vec![drop_default],
+            })
+        }
+    }
+
+    fn parse_alter_table_set_default(&mut self, column_name: Ident) -> Result<AlterTableOperation> {
+        let default_constraint = self.parser.parse_expr().context(error::SyntaxSnafu)?;
+        let set_default = SetDefaultsOperation {
+            column_name,
+            default_constraint,
+        };
+        if self.parser.consume_token(&Token::Comma) {
+            let mut defaults = self
+                .parser
+                .parse_comma_separated(parse_alter_column_set_default)
+                .context(error::SyntaxSnafu)?;
+            defaults.insert(0, set_default);
+            Ok(AlterTableOperation::SetDefaults { defaults })
```

Comment:
Supporting multiple set/unset default in one alter statement makes the parser complicated. Maybe we can only support one set/unset in one alter statement?